cms/static/js/views/xblock_string_field_editor.js

```diff
@@ -30,9 +31,10 @@ define(['js/views/baseview', 'js/views/utils/xblock_utils'],
 
             render: function() {
                 this.$el.append(this.template({
-                    value: this.model.escape(this.fieldName),
+                    value: this.model.get(this.fieldName),
```

Comment:
I was supposed to scope this to the LMS, so I may omit this file entirely to retain the scope. We'll tackle all of this in 45 days O.o